src/components/behandlerdialog/meldingtilbehandler/MeldingTilBehandler.tsx

```diff
@@ -16,15 +18,26 @@ const MeldingTilBehandlerAlert = styled(Alert)`
 `;
 
 export const MeldingTilBehandler = () => {
+  const { isFeatureEnabled } = useFeatureToggles();
+  const isBehandlerdialogLegeerklaringEnabled = isFeatureEnabled(
+    ToggleNames.behandlerdialogLegeerklaring
+  );
+
   return (
     <>
       <Heading level="1" size="large" spacing>
         {texts.header}
       </Heading>
-      <MeldingTilBehandlerAlert variant="warning" size="small">
-        {texts.alert}
-      </MeldingTilBehandlerAlert>
-      <MeldingTilBehandlerSkjema />
+      {!isBehandlerdialogLegeerklaringEnabled && (
+        <MeldingTilBehandlerAlert variant="warning" size="small">
+          {texts.alert}
+        </MeldingTilBehandlerAlert>
+      )}
```

Comment:
Vi gjør slikt fordi denne må fortsatt vises for de som ikke har togglen om legeerklæring på.